packages/console/src/pages/Applications/components/GetStarted/index.tsx

```diff
@@ -20,10 +24,57 @@ type Props = {
   onClose: () => void;
 };
 
+type DocumentFileNames = {
+  files: string[];
+};
+
+type Step = {
+  title?: string;
+  subtitle?: string;
+  metadata: string;
+};
+
 const GetStarted = ({ appName, onClose }: Props) => {
   const [isLibrarySelectorFolded, setIsLibrarySelectorFolded] = useState(false);
-  const [libraryName, setLibraryName] = useState<string>(SupportedJavascriptLibraries.Angular);
+  const [libraryName, setLibraryName] = useState<string>(SupportedJavascriptLibraries.React);
+  const [steps, setSteps] = useState<Step[]>();
```

Comment:
nit: could give a default empty []? might avoid the following not empty check

Reply:
Yeah, but this is also fine. I prefer undefined